Formula/infer.rb

```diff
@@ -127,8 +95,30 @@ def install
       }
     EOS
 
-    shell_output("#{bin}/infer --fail-on-issue -P -- clang -c FailingTest.c", 2)
-    shell_output("#{bin}/infer --fail-on-issue -P -- clang -c PassingTest.c")
+    no_issues_output =
+      ["", "  No issues found  ", ""].join("\n")
```

Comment:
```suggestion
    no_issues_output = "\n  No issues found  \n"
```
Note that `assert_match` tests whether there is a matching substring, so this should probably work too:
```ruby
no_issues_output = "No issues found"
```
However, `assert_equal` is better than `assert_match` wherever possible.